SwiftLeeds/Views/Common/SectionHeader.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CommonCompo.swift
+//  SwiftLeeds
+//
+//  Created by LUCKY AGARWAL on 23/07/22.
+//
+
+import SwiftUI
+
+struct SectionHeader: View {
+    let title: String
+    let fontStyle: Font
+    let foregroundColor: Color
+    let maxWidth: CGFloat
+    let alignment: Alignment
+    let accesibilityAddTraits: AccessibilityTraits
+    
+    internal init(_title: String,
```

Comment:
Not sure why this is using _title instead of just title, then using self.title = title inside the initialiser